### ISSUE TITLE
New canvas file extension cannot be other than .kra

### ISSUE THREAD

**Issue by firmart** [lepisma/org-krita#4]
https://github.com/lepisma/org-krita/blob/720c4e70a29fb2c8b6c76e44e2f397d0d82eafae/org-krita.el#L100

The line above is doubtful. This is why:

1. `org-krita-insert-new-image` ask for a filepath (without specifying any constraints). Suppose the user fed it `/path/to/myimage`, then `resources/template.kra` is copied to `/path/to/myimage`.
2. The line 100 try to compute the `.kra` path by substituting the extension to `.kra` (or appending `.kra` if there is no extension). Consequently,  `(expand-file-name (f-swap-ext "/path/to/myimage.kra" "kra"))` evals to `"/path/to/myimage.kra"`. Then, `krita` try to open  `"/path/to/myimage.kra"` which doesn't exist. 

Conclusion:
- If the filepath fed doesn't have an extension or have another than `.kra`, then it's not possible to edit the canvas.

I'm not sure why the extension should be added/substituted (maybe for windows ?), but at least the file should be renamed accordingly & this behaviour should be documented somewhere (e.g. prompted).

**Comment by lepisma** [comment]
Ok, I see why this was kept this way.

Earlier there was a separate preview file (a `png`) that was exported from the `kra` file. I see the issue.

I changed the system to prompt the user when the file name is not having the .kra extension. The UX can be improved a bit more but let me know what you think.

**Comment by firmart** [comment]
I think that is better to have less user interaction. Users probably won't input the `.kra` extension. This leads to 3 prompts which is too much in my opinion. I moved this option to customization in #6. In fact, I'm thinking about adding two more customizations to get rid of the two first arguments (if users want).

**Comment by firmart** [comment]
I added two customizations in #6. Maybe there is a better way to do it, but the idea is to offer the possibility to create an image without being prompted. Besides drawing, another use case of `org-krita` is note-taking. In such a circumstance, people may not care about file-path & description. For instance, I made my own function which put `.kra` files in an assets directory with automatically numbering, and I define `org-krita-get-new-desc` as an alias of `org-krita-get-new-filepath`.
